src/Compilers/Core/Portable/Compilation/IExpression.cs

```diff
@@ -795,31 +795,21 @@ public interface IObjectCreationExpression : IExpression
         /// <summary>
         /// Explicitly-specified member initializers.
         /// </summary>
-        ImmutableArray<IMemberInitializer> MemberInitializers { get; }
+        ImmutableArray<ISymbolInitializer> MemberInitializers { get; }
     }
 
     /// <summary>
     /// Represents an object member initializer.
     /// </summary>
-    public interface IMemberInitializer : IOperation
+    public interface ISymbolInitializer : IOperation
     {
-        MemberInitializerKind MemberInitializerKind { get; }
         IExpression Value { get; }
     }
-
-    /// <summary>
-    /// Kinds of member initializers.
-    /// </summary>
-    public enum MemberInitializerKind
-    {
-        Field,
-        Property
-    }
-
+    
     /// <summary>
     /// Represents an initialization of a field.
     /// </summary>
-    public interface IFieldInitializer : IMemberInitializer
+    public interface IFieldInitializer : ISymbolInitializer
```

Comment:
Within the VB compiler, there is logic for handling multiple properties that share an initializer in addition to multiple fields that share an initializer. Is it actually possible to write VB code in which two properties share an initializer?